### DISCUSSION TITLE
Versioning in custom modules...

### DISCUSSION THREAD

**Discussion by ThiloSO** [joomla/joomla-cms#35315] [2. Cold Features]
Feature Request:

The nice function "versions" which is accessible through each article would also be great for custom modules...

Any thoughts about that?

Greets

**Comment by PhilETaylor** [comment]
Currently Modules do not implement the Versionable trait and therefore this is currently not possible. 

This should be moved to [Discussions](https://github.com/joomla/joomla-cms/discussions) as is a feature request and not a bug.

**Comment by brianteeman** [comment]
This has been asked for before. If I remember part of the reason why it was not implemented (yet?) was that it would be confusing to have versions on this module but not on every module.

#11136
